### PR TITLE
Gas usage of CALL should first take C_CALLGAS into account

### PR DESCRIPTION
**Pull request by pirapira**
Before this change, when the CALL instruction got gas refuned from the callee, the caller didn't losing the initial gas passed to the callee.
This change makes the caller to lose the initial gas passed to the callee before receiving the remainder.

### CODE REVIEW

**Comment by pirapira** [comment]
This is a reincarnation of #304 (the first commit).  The second commit of #304 goes to `tangerine-whistle`.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- sudo apt-get -qq update
+- sudo apt-get install texlive texlive-latex3
+script:
+- ./build.sh
+deploy:
+  provider: script
+  script: ./travis_deploy.sh
+  skip_cleanup: true
+  on:
+    branch: master
```
Comment (nicksavers) [inline .travis.yml]: If you merge this into `frontier`, the branch for the script should also use that branch.

Comment (pirapira) [reply .travis.yml]: Really?  I don't want to deploy PDFs from `frontier`.

**Comment by pirapira** [comment]
Since I have
```
on: master
```

The PDF will not be deployed when you merge this into `frontier`.  See the Travis document on this:
https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A

**Comment by pirapira** [comment]
I prefer to have identical `.travis.yml` on all branches.

**Comment by pirapira** [comment]
@nicksavers can we move ahead here?

**Pull request merged**